catalog/transformer/catalog-transformer-pdf/src/test/java/ddf/catalog/transformer/input/pdf/GeoPdfDocumentGenerator.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.input.pdf;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.pdfbox.cos.COSArray;
+import org.apache.pdfbox.cos.COSDictionary;
+import org.apache.pdfbox.cos.COSString;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+
+public class GeoPdfDocumentGenerator {
+
+    public static PDDocument generateEmptyPdf() {
+        PDDocument pdDocument = new PDDocument();
+        pdDocument.addPage(new PDPage());
+        return pdDocument;
+    }
+
+    public static PDDocument generateGeoPdf(int numberOfPages, int numberOfFramesPerPage,
+            String projectionType) {
+        PDDocument pdDocument = new PDDocument();
+        for (int i = 0; i < numberOfPages; i++) {
```

Comment:
Since this is only for testing, I'm going to keep it as a for-loop for simplicity